cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -141,7 +144,19 @@ func (p *Provisioner) Delete(pv *v1.PersistentVolume) (err error) {
 		}
 		return p.DeleteHostPath(pv)
 	}
-
+	sendEventOrIgnore(pv.Name, GetLocalPVType(pv), analytics.LocalPVDelete)
 	glog.Infof("Retained volume %v", pv.Name)
 	return nil
 }
+
+func sendEventOrIgnore(name, storageType, method string) {
```

Comment:
Need to explain in code comments when this should get invoked. Based on that i can comment the rest part

Reply:
Done. This is same as in the `volume_endpoint_v1alpha1.go`